tests/test_browser.py

```diff
@@ -1247,6 +1247,7 @@ def test_webgl_context_attributes(self):
     # perform tests with attributes activated
     self.btest('test_webgl_context_attributes_glut.c', '1', args=['--js-library', 'check_webgl_attributes_support.js', '-DAA_ACTIVATED', '-DDEPTH_ACTIVATED', '-DSTENCIL_ACTIVATED', '-DALPHA_ACTIVATED', '-lGL', '-lglut', '-lGLEW'])
     self.btest('test_webgl_context_attributes_sdl.c', '1', args=['--js-library', 'check_webgl_attributes_support.js', '-DAA_ACTIVATED', '-DDEPTH_ACTIVATED', '-DSTENCIL_ACTIVATED', '-DALPHA_ACTIVATED', '-lGL', '-lSDL', '-lGLEW'])
+    self.btest('test_webgl_context_attributes_sdl2.c', '1', args=['--js-library', 'check_webgl_attributes_support.js', '-DAA_ACTIVATED', '-DDEPTH_ACTIVATED', '-DSTENCIL_ACTIVATED', '-DALPHA_ACTIVATED', '-lGL', '-s', 'USE_SDL=2', '-lGLEW'])
```

Comment:
is that a new file? maybe forgot to add it to the PR. but also, should it have `egl` in the name?

Reply:
-_-'
That's a new file. I decided to run the test at the SDL2 level, which is what people more commonly use, and which itself relies on EGL.

Reply:
Oh, I see, so this PR fixes SDL2 as well?

Reply:
Yes, that's the goal :)